docusaurus.config.js

```diff
@@ -36,7 +36,13 @@ const config = {
       ({
         docs: {
           sidebarPath: require.resolve("./sidebars.js"),
-          editUrl: "https://github.com/brimdata/zed/tree/main",
+          editUrl: ({version, docPath}) => {
+            if (version === "current") {
+              return `https://github.com/brimdata/zed/edit/main/docs/${docPath}`;
+            } else {
+              return `https://github.com/brimdata/zed/tree/${version}/docs/${docPath}`;
+            }
+          },
```

Comment:
Idk since a versioned document is from a frozen commit, I'm kind of thinking the edit button should just link to the current doc page on the main branch since these are the changes we would be interested in seeing. If that page happens to not exist, oh well. I don't see us re-cutting a previous release based on a fix to documentation. Indeed the docusaurus doc site appears to take the same approach (see "edit page" link at the bottom of https://docusaurus.io/docs/2.0.0-beta.22/versioning).